app/models/question.rb

```diff
@@ -36,14 +36,18 @@ def starred_by?(user)
     evaluation_by(:stars, user) == 1
   end
 
+  def expired?
```

Comment:
I agree with @samnang that it's better to add a boolean column instead. But let's name it `closed`.